### PR TITLE
add dockerfile and chart for new stats

### PR DESCRIPTION
**Pull request by hossnys**
### Description

adding new-stats package Dockerfile , chart , and workflows for building docker docker image .

### Changes

- add Dockerfile 
- add chart.yaml
- add values.yaml 
- add build workflow
- add docker image workflow 

### Related Issues

https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1281

### CODE REVIEW

**Review by zaelgohary — COMMENTED** [review]
I think this package should be named stats and the old one's name should be changed to old-stats till it's archived.

**Comment by hossnys** [comment]
> I think this package should be named stats and the old one's name should be changed to old-stats till it's archived.

we can keep the current naming till we get rid off the old one , then we rename it in next version as to avoid getting any of old images deployed .

**Review by 0oM4R — COMMENTED** [review]
### update not an issue; was running docker build in the new_stats directory
image
I tried to build the image locally and got that error

File: .github/workflows/new_stats_build.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          cache-dependency-path: "**/yarn.lock"
+
+      - name: Install dependencies
+        run: |
+          yarn install
+
+      - name: Build
+        run: |
+          lerna run build --no-private
+          yarn workspace @threefold/newstats build
```
Comment (0oM4R) [inline .github/workflows/new_stats_build.yaml:45]: i believe it should be `newstats` only without `@threefold`

File: packages/new_stats/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+FROM node:18 AS build
+
+WORKDIR /app
+
+COPY . /app/
+
+# Install yarn and build project
+RUN yarn install
+RUN yarn lerna run build --no-private && yarn workspace @threefold/newstats build
```
Comment (0oM4R) [inline packages/new_stats/Dockerfile:9]: same here docker build fails on this step 'error Unknown workspace "@threefold/newstats".'

**Review by 0oM4R — CHANGES_REQUESTED** [review]

File: packages/new_stats/Dockerfile (hunk 2 above)
Comment (AhmedHanafy725) [reply packages/new_stats/Dockerfile:9]: any package should be prefixed with `@threefold` in the sdk, So we should fix the package name

File: .github/workflows/new_stats_build.yaml (hunk 1 above)
Comment (hossnys) [reply .github/workflows/new_stats_build.yaml:45]: as per @AhmedHanafy725 comment ,this prefix should be added from your side

**Review by 0oM4R — APPROVED** [review]

**Review by amiraabouhadid — APPROVED** [review]

**Pull request merged**